chain/client/tests/process_blocks.rs

```diff
@@ -43,6 +43,8 @@ use near_primitives::sharding::{EncodedShardChunk, ReedSolomonWrapper, ShardChun
 #[cfg(feature = "protocol_feature_block_header_v3")]
 use near_primitives::sharding::{ShardChunkHeaderInner, ShardChunkHeaderV3};
 
+#[cfg(feature = "protocol_feature_fix_storage_usage")]
```

Comment:
Generally feel that we somewhat abuse Rust features here, don't we?

Reply:
I moved this test to its own mod to simplify imports